package.json

```diff
@@ -13,19 +13,19 @@
         "prepare": "patch-package"
     },
     "dependencies": {
-        "@ecl/builder": "2.31.0",
-        "@ecl/ec-preset-editor": "2.31.0",
-        "@ecl/ec-preset-legacy-website": "2.31.0",
-        "@ecl/ec-component-form-help-block": "2.31.0",
-        "@ecl/ec-theme-default": "2.31.0",
-        "@ecl/eu-preset-legacy-website": "2.31.0",
-        "@ecl/eu-theme-default": "2.31.0",
-        "@ecl/eu-component-form-help-block": "2.31.0",
+        "@ecl/builder": "2.32.0",
+        "@ecl/ec-preset-editor": "2.32.0",
+        "@ecl/ec-preset-legacy-website": "2.32.0",
+        "@ecl/ec-component-form-help-block": "2.32.0",
+        "@ecl/ec-theme-default": "2.32.0",
+        "@ecl/eu-preset-legacy-website": "2.32.0",
+        "@ecl/eu-theme-default": "2.32.0",
+        "@ecl/eu-component-form-help-block": "2.32.0",
         "@ecl/stylelint-config": "0.2.1",
-        "@ecl/ec-component-fact-figures": "2.31.0",
-        "@ecl/ec-component-media-container": "2.31.0",
-        "@ecl-twig/ec-components": "2.31.1",
-        "node-sass": "4.13.1",
+        "@ecl/ec-component-fact-figures": "2.32.0",
+        "@ecl/ec-component-media-container": "2.32.0",
+        "@ecl-twig/ec-components": "2.32.0",
+        "node-sass": "4.12.0",
```

Comment:
This needs to be back to "node-sass": "4.13.1", to suppress GitHub security warnings.